packages/react-aria-components/docs/Table.mdx

```diff
@@ -1103,6 +1114,7 @@ function Example() {
 
 .react-aria-DropIndicator[data-drop-target] {
   outline: 1px solid var(--highlight-background);
+  transform: translateZ(0);
```

Comment:
drop indicator changes fix issue where drop indicator would not stretch entire width of safari tables